network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html

```diff
@@ -86,6 +86,40 @@ <h2 class="h4-heading">{% trans "Trending projects" %}</h2>
     <a class="cta-link d-block d-md-none" href="https://www.mozillapulse.org/add">{% trans "Add your project" %}</a>
   </div>
 </div>
+
+</div> <!-- end .container -->
+{% image page.partner_background_image fill-1200x600 as partner_img %}
+<div style='background-image: url("{{ partner_img.url }}"); background-size: cover; background-position: center center; color: #fff !important;' class="py-5">
```

Comment:
let's not do the frontend work in this ticket: setting up the HTML skeleton for this section is fine, but let's not do inline-styling (this code is going to have to be in the SCSS files, with that background pulled from the static images dir)

Reply:
This image is a media file uploaded by the user, so we'll need some way to work with that and crop it using Wagtails `{% image %}` tag. For now I'll remove all the other inline CSS but keep the background image so FE knows what to work with, how's that sound?